### PR TITLE
Add v1 code back

### PR DESCRIPTION
**Pull request by ayousuf23**
# PR Summary

This PR adds back the v1 code, tests, build scripts, and yaml files to the `Archive_v1` folder. It updates the required PowerShell version for the v2.0.1 module to PowerShell v7.3.0

## PR Context

See above.

### CODE REVIEW

**Comment by daxian-dbw** [comment]
@ayousuf23 Thanks for getting to this promptly!
Can you please add the build scripts and ymal files related to v1 to the Archive_v1 folder as well?

**Comment by ayousuf23** [comment]
@daxian-dbw Sure, I'll add those as well

**Comment by ayousuf23** [comment]
@daxian-dbw I added the build scripts and yaml files. I'm waiting for the tests to run and then it can be merged.

**Comment by ayousuf23** [comment]
@daxian-dbw The tests passed. You can merge it now

**Comment by ayousuf23** [comment]
I don't have permission to merge.

**Comment by daxian-dbw** [comment]
> I don't have permission to merge.

Me neither 😄. @SteveL-MSFT can you please merge?

**Comment by iSazonov** [comment]
Why not preserve a clean history and create new repository PowerShell/Microsoft.PowerShell.Archive.**V2**?

**Comment by SeeminglyScience** [comment]
Talked this over with everyone and we're gonna go with creating a branch off the old code in case it's needed for servicing.

**Comment by SeeminglyScience** [comment]
FYI https://github.com/PowerShell/Microsoft.PowerShell.Archive/tree/servicing

File: src/Microsoft.PowerShell.Archive.psd1 [hunk 1]
```diff
@@ -5,7 +5,7 @@ Author = 'Microsoft'
 CompanyName = 'Microsoft'
 Copyright = '(c) Microsoft. All rights reserved.'
 Description = 'PowerShell module for creating and expanding archives.'
-PowerShellVersion = '7.2.5'
+PowerShellVersion = '7.3.0'
```
Comment (daxian-dbw) [inline src/Microsoft.PowerShell.Archive.psd1:8]: @ayousuf23 Please open a new PR for this version change.

**Pull request closed without merging**